### PR TITLE
Update RTOS.md

### PR DESCRIPTION
**Pull request by AnotherButler**
Separate RTOS and event config parameters to show section separation.

### CODE REVIEW

File: docs/reference/configuration/RTOS.md [hunk 1]
```diff
@@ -19,7 +21,11 @@ Name: rtos.present
     Defined by: library:rtos
     Macro name: MBED_CONF_RTOS_PRESENT
     Value: 1 (set by library:rtos)
+```
 
+### Event configuration parameters
```
Comment (bulislaw) [inline docs/reference/configuration/RTOS.md]: That's "Event queue" https://os.mbed.com/docs/v5.9/reference/eventqueue.html

Comment (AnotherButler) [reply docs/reference/configuration/RTOS.md]: It's not "event": https://os.mbed.com/docs/v5.9/reference/event.html? That's confusing

Comment (bulislaw) [reply docs/reference/configuration/RTOS.md]: Event class represents and event in the event queue. The config options refer to the settings of EventQueue.

**Comment by AnotherButler** [comment]
@bulislaw Is this OK to merge now?

**Comment by AnotherButler** [comment]
@jen3andruska Could you please do the honors and merge this at your convenience?

**Comment by jen3andruska** [comment]
@AnotherButler Sounds great, will do!

**Pull request merged**